bbot/modules/ajaxpro.py

```diff
@@ -0,0 +1,49 @@
+import re
+from bbot.modules.base import BaseModule
+
+
+class ajaxpro(BaseModule):
+    """
+    Reference: https://mogwailabs.de/en/blog/2022/01/vulnerability-spotlight-rce-in-ajax.net-professional/
+    """
+
+    ajaxpro_regex = re.compile(r'<script.+src="([\/a-zA-Z0-9\._]+,[a-zA-Z0-9\._]+\.ashx)"')
+    watched_events = ["HTTP_RESPONSE", "URL"]
+    produced_events = ["VULNERABILITY", "FINDING"]
+    flags = ["active", "safe", "web-thorough"]
+    meta = {"description": "Check for potentially vulnerable Ajaxpro instances"}
+
+    async def handle_event(self, event):
+        if event.type == "URL":
+            probe_url = f"{event.data}ajaxpro/whatever.ashx"
```

Comment:
@liquidsec  this should probably handle cases where the URL doesn't end in a slash. Or instead we could add a filter_event that only accepts dirs.

Reply:
added a check in the URL side to require a 'dir' tag. Couldn't put it in a filter because we want the HTTP_RESPONSE side to look at every URL event.